mmv1/templates/terraform/examples/edgecontainer_local_control_plane_node_pool_internal.tf.erb

```diff
@@ -23,7 +23,6 @@ resource "google_edgecontainer_cluster" "cluster" {
     local {
       node_location = "us-central1-edge-den25349"
       node_count = 1
-      machine_filter = "den25349-01"
```

Comment:
Will this field be present in any test if it's removed here?

Reply:
This field will not exist in other tests. This is intended as we don't wan the test only runs on specific machines.